docs/conf.py

```diff
@@ -49,7 +49,7 @@
 author = 'The Trieste Contributors'
 
 # The full version, including alpha/beta/rc tags
-release = 'v0.1.0'
+release = 'v0.2.0'
```

Comment:
we're uploading to the already existent trieste pypi package. That has versions 0.1.0 and 0.1.1, so bumping to 0.2.0 clearly delineates this from the old trieste project. We may choose to delete the 0.1.0 and 0.1.1 versions